frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx

```diff
@@ -105,8 +105,7 @@ export const PopinFixedHeader = (props) => {
   const filteredHeaderButtons = headerButtons ? headerButtons.filter(action => action.showAction) : []
 
   return (
-    <div className={editTitle
-      ? classnames('wsContentGeneric__header', `${customClass}__header__isEditing`, 'wsContentGeneric__header__isEditing') : classnames('wsContentGeneric__header', `${customClass}__header`)}
+    <div className={classnames('wsContentGeneric__header', `${customClass}__header${editTitle ? '__isEditing' : ''}`, {'wsContentGeneric__header__isEditing': editTitle })}
```

Comment:
It works and uses classnames features but it can be simpler

```suggestion
<div className={classnames(
    'wsContentGeneric__header',
    {
        `${customClass}__header`: !editTitle,
        `${customClass}__header__isEditing`: editTitle,
        'wsContentGeneric__header__isEditing': editTitle,
    }
)}>
```

Reply:
this suggestion didn't compile

Reply:
done